src/Http/Http.Features/src/ISession.cs

```diff
@@ -43,7 +44,7 @@ public interface ISession
         /// <param name="key"></param>
         /// <param name="value"></param>
         /// <returns></returns>
-        bool TryGetValue(string key, out byte[] value);
+        bool TryGetValue(string key, [MaybeNullWhen(false)] out byte[] value);
```

Comment:
I'll see if I can port this change to 5.0. It is more restrictive so we might as well try and get it right.

Reply:
Ideally we would have:

```suggestion
        bool TryGetValue(string key, [NotNullWhen(true)] out byte[]? value);
```

Reply:
> This might be more problematic with how distributed session works. I think it can return null while returning true.

I wouldn't expect that. Where do you see it happening?